README.md

```diff
@@ -190,6 +190,31 @@ but not the `nats` section. Set `route_mode` to 'tcp'.
 If both modes are required, the `oauth`, `routing_api`, and `nats` sections are required.
 Set `route_mode` to 'all'.
 
+Service Broker
+--------------
+
+If per route config is required the controller can be ran as a [service broker.](https://docs.cloudfoundry.org/services/overview.html) Running as a service
+broker allows control over which BIG-IP objects are applied to a particular route through plans. For more information see [F5 BIG-IP Controller for Cloud Foundry docs](http://clouddocs.f5.com/products/connectors/cf-bigip-ctlr/latest/).
```

Comment:
Remove everything but the line I provided above to prevent duplication of content.